src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,297 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+
+#include <boost/algorithm/string.hpp>
+
+#include <unordered_map>
 
 namespace ripple {
 
+namespace detail {
+
+class ServerDefinitions
+{
+private:
+    std::string const
+    translate(std::string const& inp);
+
+    Json::Value
+    generate();
+
+    uint256 defsHash_;
+    Json::Value defs_;
+
+public:
+    ServerDefinitions() : defs_(generate()){};
+
+    bool
+    hashMatches(uint256 hash) const
+    {
+        return defsHash_ == hash;
+    }
+
+    Json::Value const&
+    operator()() const
+    {
+        return defs_;
+    }
+};
+
+std::string const
+ServerDefinitions::translate(std::string const& inp)
+{
+    auto replace = [&](const char* oldStr, const char* newStr) -> std::string {
```

Comment:
Nit: `const` is in the wrong place (should be `char const*`)

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/b628b4203dd54345ce73597005be6a3e1727be7f